rustls-test/src/lib.rs

```diff
@@ -471,16 +471,19 @@ impl KeyType {
         .unwrap()
     }
 
-    pub fn ca_distinguished_name(&self) -> &'static [u8] {
+    pub fn ca_distinguished_name(&self) -> Vec<u8> {
         match self {
-            Self::Rsa2048 => b"0\x1f1\x1d0\x1b\x06\x03U\x04\x03\x0c\x14ponytown RSA 2048 CA",
+            Self::Rsa2048 => {
+                b"0\x1f1\x1d0\x1b\x06\x03U\x04\x03\x0c\x14ponytown RSA 2048 CA" as &'static [u8]
```

Comment:
is `&b"abc"[..]` more idiomatic?

Reply:
I guess it might be! At least it's more concise/gets better formatting.